modules/lib/lib-project/src/main/resources/lib/xp/examples/project/modify.js

```diff
@@ -0,0 +1,44 @@
+// BEGIN
+// Modifies an existing content project
+var projectLib = require('/lib/xp/project');
+
+var project = projectLib.modify({
+    id: 'my-project',
+    displayName: 'New project name',
+    description: 'New project description',
+    language: 'en'
+});
+// END
+
+// BEGIN
+// Modifies an existing content project inside context with `system.admin` role
+var projectLib = require('/lib/xp/project');
+var contextLib = require('/lib/xp/context');
+
+var modifyProject = function () {
+    return projectLib.modify({
+        id: 'my-project',
+        displayName: 'New project name',
+        description: 'New project description',
+        language: 'en'
+    });
+}
+
+var project = contextLib.run({
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['project' is already defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5519867)